server/controllers/profile.js

```diff
@@ -0,0 +1,54 @@
+import express from 'express';
+import mongoose from 'mongoose';
+
+import Profile from '../models/Profile.js';
+
+const router = express.Router();
+
+//GET ALL PROFILES
+export const getAllProfiles = async (req, res) => {
+    try {
+        const profiles = await Profile.find();
+        res.status(200).json(profiles);
+    } catch (err) {
+        res.status(500).json(err);
+    }
+}
+
+//GET PROFILE BY ID
+export const getProfile = async (req, res) => {
+    try {  
+        const profile = await Profile.findOne(req.params.id);
+        res.status(200).json(profile);
+    } catch (err) {
+        res.status(500).json(err);
+    }
+}
+
+//CREATE PROFILE
+export const createProfile = async (req, res) => {
+    const newProfile = new Profile(req.body);
```

Comment:
It is a good practice to destructure the request body to only allow the required fields